diff --git a/node_modules/xstate/lib/interpreter.d.ts b/node_modules/xstate/lib/interpreter.d.ts
index 1aec402..2daf5ec 100644
--- a/node_modules/xstate/lib/interpreter.d.ts
+++ b/node_modules/xstate/lib/interpreter.d.ts
@@ -168,7 +168,6 @@ export declare class Interpreter<TContext, TStateSchema extends StateSchema = an
     toJSON(): {
         id: string;
     };
-    [Symbol.observable](): this;
 }
 export declare function spawn<TC, TE extends EventObject>(entity: StateMachine<TC, any, TE>, nameOrOptions?: string | SpawnOptions): Interpreter<TC, any, TE>;
 export declare function spawn(entity: Spawnable, nameOrOptions?: string | SpawnOptions): Actor;
